src/ecrc-frontend/src/components/page/consent/Consent.js

```diff
@@ -21,7 +21,7 @@ export default function Consent({ page: { header }, onContinueClick }) {
   useEffect(() => {
     if (firstBoxChecked && secondBoxChecked && thirdBoxChecked) {
       setInputEnabled("textinput_editable_white");
-      if (applicantName) {
+      if (applicantName !== "") {
```

Comment:
why was this changed? did you need to explicitly check for nulls?

Reply:
no, no, wrong branch. so, something is wrong.
will change back.